test/HubPool.TokenWhitelisting.ts

```diff
@@ -0,0 +1,38 @@
+import { expect } from "chai";
```

Comment:
notice that I've separated sets of tests into files. goal is to not have one massive file and rather just run some fixtures and have fewer shorter test files.